docs/working-with-data.md

```diff
@@ -0,0 +1,43 @@
+# Working with Data
+
+BC Data Service develops data solutions that help government better serve the people of B.C.
+
+The division is the trusted go-to leader for data analysis and statistics about people in B.C. It leads data policy development and management and also provides data analysis, tools, platforms, training and support.
+
+## API Resources
+
+### API Directory
+
+The [API Directory](https://api.gov.bc.ca/devportal/api-directory) provides a place to share your API for developers to discover.
+
+### API Management Platform
+
+The BC Government [API Management Platform](docs/default/component/aps-infra-platform-docs) is a common component that serves as a central entry point for APIs, and acts as an intermediary between applications and backend services. 
+
+### API Registry
+
+APIs within the [BC Data Catalogue](#bc-data-catalogue) can be found in the [API Registry](https://catalogue.data.gov.bc.ca/group/bc-government-api-registry)
+
+### API Services Portal
+
+If you are a developer or part of a government team, you can administer and manage your APIs using the [API Services Portal](https://api.gov.bc.ca/).
+
+## Other Resources
+
+### BC Data Catalogue
+
+[The BC Data Catalogue](https://www2.gov.bc.ca/gov/content/data/bc-data-catalogue) provides the easiest access to government's data holdings, as well as their associated applications and web services. Thousands of the datasets discoverable in the Catalogue are available under the [Open Government License - British Columbia](https://www2.gov.bc.ca/gov/content/data/open-data/open-government-licence-bc).
+
+### Geospatial Data
+
+The province's area, varied topography and predominantly natural resource based economy has positioned B.C. as a leader in geographic information management and generates a wealth of geographic information that is discoverable in the BC Data Catalogue.  BC Data Service provides access to this information through a suite of unique applications and services that comprise [the B.C. Spatial Data Infrastructure](https://www2.gov.bc.ca/gov/content/data/geographic-data-services).
+
+## Standards
+
+* [API Guidelines](https://classic.developer.gov.bc.ca/Data-and-APIs/BC-Government-API-Guidelines)
```

Comment:
Can we move this further up, at the end of 'Work with Data' section? I feel standards should always be "front and centre".